util/reggen/reg_block.py

```diff
@@ -507,3 +569,86 @@ def has_internal_shadowed_reg(self) -> bool:
                 return True
 
         return False
+
+    def apply_alias(self, alias_block: object, where: str) -> None:
```

Comment:
As before, I think the type of `alias_block` can probably be tightened up.

Reply:
Jep, done!